crates/model/src/defi/dex.rs

```diff
@@ -96,3 +99,46 @@ impl Display for Dex {
         write!(f, "Dex(chain={}, name={})", self.chain, self.name)
     }
 }
+
+impl From<Pool> for CurrencyPair {
+    fn from(p: Pool) -> Self {
+        let symbol = Symbol::from(format!("{}/{}", p.token0.symbol, p.token1.symbol));
+        let id = InstrumentId::new(symbol, Venue::from("DEFI"));
```

Comment:
I think the venue should probably be the [identifier for the DEX](https://github.com/nautechsystems/nautilus_trader/blob/develop/crates/model/src/defi/dex.rs#L94).

Reply:
done, thx